opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -944,6 +944,35 @@ def test_add_link_with_invalid_span_context(self):
 
             self.assertEqual(len(root.links), 0)
 
+    def test_add_link_with_invalid_span_context_record(self):
```

Comment:
nit: Maybe the test name is not clear enough, you can rename the test function with a more descriptive name.
I notice you are also trying multiple statements within the test, perhaps splitting it into more tests should improve the readability.

Reply:
It makes sense. I'll rename the test and split it into test with attributes and test with tracestate